src/python/gudhi/representations/vector_methods.py

```diff
@@ -912,28 +918,29 @@ def fit(self, X, y=None):
 
     def transform(self, X):
         """
-        Compute the persistence landscape for each persistence diagram individually and concatenate the results.
+        Compute the persistence lengths for each persistence diagram individually and concatenate the results.
 
         Parameters:
             X (list of n x 2 numpy arrays): input persistence diagrams.
-    
+
         Returns:
             numpy array with shape (number of diagrams) x (num_lengths): output persistence lengths.
```

Comment:
That would be nice, but isn't the current output a list?

Reply:
You are right, so I took the opportunity to rewrite it with a first result array filled with zeros, that is instantiated rows by rows on [3bbd3af](https://github.com/GUDHI/gudhi-devel/pull/1117/commits/3bbd3af834902e79e83b990ae246dc9f1469e2d7)